microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -50,6 +50,8 @@ The Evergreen Runtime is preinstalled onto all Windows 11 devices as a part of t
 <!-- ====================================================================== -->
 ## The Fixed Version runtime distribution mode
 
+The Fixed Version runtime distribution model is also called a "self-contained" distribution approach; the redistributable bits are included with your app.  This approach is similar to .NET's "Self-Contained deployment option".<!--todo: can add other examples of a Self Contained approach -->
```

Comment:
```suggestion
The Fixed Version runtime distribution mode is also called a "self-contained" distribution model; the redistributable bits are included with your app.  This approach is similar to .NET's "Self-Contained deployment option".<!--todo: can add other examples of a Self Contained approach -->
```
@liminzhu @david-risney @aluhrs13 - 
* Change "model" to "mode" here?  The live article has 29 hits on "mode", 0 hits on "model".
* Change "approach" to "model" here?

See suggested wording in GitHub Issue https://github.com/MicrosoftDocs/edge-developer/issues/2475

Rendered article section for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-2793#the-fixed-version-runtime-distribution-mode

Reply:
changing to "mode" like the other 26 hits on that phrase, in this article.  0 hits on "distribution model" in this article.